antarest/study/service.py

```diff
@@ -984,6 +985,68 @@ def create_study(
         logger.info("study %s created by user %s", raw.id, params.get_user_id())
         return str(raw.id)
 
+    def create_external_study(
+        self,
+        study_folder: StudyFolder,
+        params: RequestParameters,
+    ) -> str:
+        """
+        Creates a study with the specified study name, version, group IDs, and user parameters.
+
+        Args:
+            study_name: The name of the study to create.
+            version: The version number of the study to choose the template for creation.
+            group_ids: A possibly empty list of user group IDs to associate with the study.
+            params:
+                The parameters of the HTTP request for creation, used to determine
+                the currently logged-in user (ID and name).
+
+        Returns:
+            str: The ID of the newly created study.
+        """
+
+        author = self.get_user_name(params)
+        now = datetime.now(timezone.utc)
+        path = str(study_folder.path)
+        folder = f"{EXTERNAL_WORKSPACE_NAME}/{path}"
+        raw = RawStudy(
+            id=str(uuid4()),
+            name=study_folder.path.name,
+            folder=folder,
+            workspace=EXTERNAL_WORKSPACE_NAME,
```

Comment:
I was wondering if having a workspace name which does not have any configuration could be an issue, but it does not seem so, by reading the use of `workspace` in the code (not used very much, apart from checking if a study is a "managed" one).
We should carefully test the behaviour though, to be sure.